### PR TITLE
Add an RPC to get the recent network usage

### PR DESCRIPTION
**Pull request by sgkim126**

### CODE REVIEW

File: spec/JSON-RPC.md [hunk 1]
```diff
@@ -2245,6 +2246,38 @@ No parameters
 
 [Back to **List of methods**](#list-of-methods)
 
+## net_recentNetworkUsage
+Gets the recent network usage.
```
Comment (ScarletBlue) [inline spec/JSON-RPC.md]: Gets the recent network usage and returns an object.

File: spec/JSON-RPC.md [hunk 2]
```diff
@@ -2245,6 +2246,38 @@ No parameters
...
+It returns an object.
+The key of the object is a meaningful separation, but what is used as a key depends on the implementation.
```
Comment (ScarletBlue) [inline spec/JSON-RPC.md]: I think this part sounds a bit awkward. Let's try fixing it.

File: spec/JSON-RPC.md [hunk 3]
```diff
@@ -2245,6 +2246,38 @@ No parameters
...
+The value of the object is the size of bytes that the node sent in the recent period.
+The exact timespan of the recent is also an implantation dependent.
```
Comment (ScarletBlue) [inline spec/JSON-RPC.md]: What is implantation dependent?

Comment (sgkim126) [reply spec/JSON-RPC.md]: Typo. implementation dependent

File: spec/JSON-RPC.md (hunk 1 above)
Comment (sgkim126) [reply spec/JSON-RPC.md]: I'll change the next sentence to 'The return type is an object.'

File: spec/JSON-RPC.md (hunk 2 above)
Comment (sgkim126) [reply spec/JSON-RPC.md]: I changed it to `The key of the object is a string, but what the keys are depend on the implementation.`

**Comment by sgkim126** [comment]
I split the commit into two commits. I'll merge the implementation first for Corgi network and update the spec later.

**Pull request merged**